### PR TITLE
Improvements to the physics engine.

### LINKED ISSUE #24: Modify object collision to work for float coordinates.

### PR DESCRIPTION
**Pull request by sammyers**
Fixes #24

### CODE REVIEW

**Comment by sammyers** [comment]
Note: some clipping into blocks still happens when colliding at high velocities.

File: src/La Trop/controller/controller.cpp [hunk 1]
```diff
@@ -10,44 +10,27 @@
...
-            _model->movePlayer(-1,0);
-            break;
-        case GLUT_KEY_DOWN:
-            _model->movePlayer(0,-1);
-            break;
-        case GLUT_KEY_RIGHT:
-            _model->movePlayer(1,0);
-            break;
-    }
+
```
Comment (segerphilip) [inline src/La Trop/controller/controller.cpp:35]: Just, completely gone?

**Review by segerphilip — APPROVED** [review]
Looks good, hopefully this doesn't break any of @hannahtwiggsmith's stuff. 🚢 🇮🇹 after testing.

File: map_example.csv [hunk 2]
```diff
@@ -0,0 +1,25 @@
+0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
```
Comment (branchwelder) [inline map_example.csv:1]: Sam4CORE

File: src/La Trop/controller/controller.cpp (hunk 1 above)
Comment (branchwelder) [reply src/La Trop/controller/controller.cpp:35]: buce

File: src/La Trop/view/view.cpp [hunk 3]
```diff
@@ -16,18 +16,21 @@ void View::render() {
...
 }
+
+void View::_renderBlock(Position position, Block block) {
+    Square square(position.first, position.second, block.getColor());
+    square.render();
+}
+
+void View::_renderPlayer(Position position) {
+    Color playerColor = (Color) { 0.75, 0.75, 0.75 };
+    Square square(position.first, position.second, playerColor);
```
Comment (branchwelder) [inline src/La Trop/view/view.cpp:34]: We're gonna wanna change this to redraw the currently existing block, right now it creates a new one.

**Review by branchwelder — COMMENTED** [review]
buce

**Pull request merged**